docs/basic.rst

```diff
@@ -67,13 +67,16 @@ A Nextflow script looks like this::
 The above example defines two processes. Their execution order is not determined by the fact that the ``blastSearch``
 process comes before ``extractTopHits`` in the script (it could also be written the other way around).
 
-Instead, because the first process defines the channel ``top_hits_ch`` in its output declarations, and the
-process ``extractTopHits`` defines the channel in its input declaration, a communication link is established.
+Instead, the pipeline (``|``) after``blastSearch`` and ``extractTopHits`` creates an anonymous channel how will serve
+as output for the fist and input for the second and a communication link is established automatically.
 
 This linking via the channels means that ``extractTopHits`` is waiting for the output of `blastSearch`, and then
 runs `reactively` when the channel has contents.
 
-.. TODO describe that both processes are launched at the same time
+In this example, when the workflow is started it will create two process and tow channels (``query_ch`` plus the anonymous)
+and it will link all of them. Both process will be started at the same time and they will be listening in their
+respective input channels. Once ``blastSearch`` start to emit values into the anonymous channel ``extractTopHits``
+will start to receiving them (i.e. ``extractTopHits`` is consuming the channel in a `reactive` way)
```

Comment:
```suggestion
In this example, when the workflow is started it will create two process and two channels (``query_ch`` plus the anonymous pipe ``|``)
and it will link all of them. Both processes will be started at the same time and they will listen to their
respective input channels. Once ``blastSearch`` starts to emit values, the anonymous channel for ``extractTopHits``
will start to receiving them (i.e. ``extractTopHits`` consumes the channel in a `reactive` way).
```

Reply:
Not sure about the *creates an anonymous channel*. Not even sure that's technically is what it's happening. 

I would say: 

> Instead, the ``|`` (pipe) operator forwards the output of the ``blastSearch`` process to the ``extractTopHits`` processes.

Reply:
Thanks Paolo, changing it